tests/SprykerEcoTest/Zed/Braintree/Business/BraintreeFacadeCheckoutPostSaveHookTest.php

```diff
@@ -0,0 +1,151 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\Braintree\Business;
+
+use Braintree\Transaction\CreditCardDetails;
+use Generated\Shared\Transfer\CheckoutResponseTransfer;
+use SprykerEco\Zed\Braintree\BraintreeConfig;
+use SprykerEco\Zed\Braintree\Business\Order\Saver;
+use SprykerEco\Zed\Braintree\Business\Payment\Transaction\PaymentTransaction;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeBridge;
+use SprykerEco\Zed\Braintree\Dependency\Facade\BraintreeToMoneyFacadeInterface;
+use SprykerEco\Zed\Braintree\Persistence\BraintreeEntityManager;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerEcoTest
+ * @group Zed
+ * @group Braintree
+ * @group Business
+ * @group Facade
+ * @group BraintreeFacadeCheckoutPostSaveHookTest
+ * Add your own group annotations below this line
+ */
+class BraintreeFacadeCheckoutPostSaveHookTest extends AbstractFacadeTest
+{
+    /**
+     * @return void
+     */
+    public function testCheckoutPostSaveHookWithSuccessfulResponse(): void
+    {
+        // Arrange
+        $response = $this->executeCheckoutPostSaveHook();
+
+        // Act
+        $result = $response->getIsSuccess();
+
+        // Assert
+        $this->assertTrue($result);
+    }
+
+    /**
+     * @return void
+     */
+    public function testCheckoutPostSaveHookWithErrorResponse(): void
+    {
+        // Arrange
+        $response = $this->executeCheckoutPostSaveHook(false);
+
+        // Act
+        $result = $response->getIsSuccess();
+
+        // Assert
+        $this->assertFalse($result);
+    }
```

Comment:
`Arrange` section - should contain preparations for test execution.
`Act` - should actually execute the testable code.
`Assert` - should contain assertions + additional methods calls e.g. to gather data for assertions from DB.  

```suggestion
    public function testCheckoutPostSaveHookWithErrorResponse(): void
    {
        // Act
        $response = $this->executeCheckoutPostSaveHook(false);

        // Assert
        $this->assertFalse($response->getIsSuccess());
    }
```

Reply:
Fixed. I know it but somehow miss this part thanks.